### PR TITLE
New package: couchdb-1.6.1

### PR DESCRIPTION
**Pull request by arete**

### CODE REVIEW

File: srcpkgs/couchdb/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'couchdb'
+pkgname=couchdb
+version=1.6.1
+revision=1
+wrksrc="apache-couchdb-$version"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --localstatedir=/var "
```
Comment (xtraeme) [inline srcpkgs/couchdb/template]: This line is redundant, those setting are set by default.

File: srcpkgs/couchdb/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+conf_files="/etc/couchdb/default.ini /etc/couchdb/local.ini"
+couchdb_homedir="/var/lib/couchdb"
+hostmakedepends="erlang"
+makedepends="libressl-devel>=2.1.4  icu-devel js-devel"
+depends="erlang icu libressl js"
+short_desc="A document-oriented database"
+maintainer="Gerardo Di Iorio <arete74@gmail.com>"
+license="APACHE"
```
Comment (xtraeme) [inline srcpkgs/couchdb/template]: Apache-2.0

File: srcpkgs/couchdb/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+version=1.6.1
+revision=1
+wrksrc="apache-couchdb-$version"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --localstatedir=/var "
+conf_files="/etc/couchdb/default.ini /etc/couchdb/local.ini"
+couchdb_homedir="/var/lib/couchdb"
+hostmakedepends="erlang"
+makedepends="libressl-devel>=2.1.4  icu-devel js-devel"
+depends="erlang icu libressl js"
```
Comment (xtraeme) [inline srcpkgs/couchdb/template]: This depends line I guess is also wrong, dependencies should be guessed automagically if the binaries are dynamically linked.

File: srcpkgs/couchdb/INSTALL [hunk 4]
```diff
@@ -0,0 +1,8 @@
+case "$ACTION" in
+        post)
+                if [ "$UPDATE" = "yes" ]; then
```
Comment (xtraeme) [inline srcpkgs/couchdb/INSTALL]: You should remove this if conditional, it makes no sense to run it only when the pkg is being updated.

**Comment by arete** [comment]
ok i have update the files

**Pull request closed without merging**